portal-api/src/drivers/fileshare.js

```diff
@@ -29,10 +25,7 @@ const getCredentials = async (fileshare = 'portal') => {
 const getShareClient = async (fileshare) => {
   const credentials = await getCredentials(fileshare);
   const { STORAGE_ACCOUNT, FILESHARE_NAME } = getConfig(fileshare);
-  const serviceClient = new ShareServiceClient(
-    `https://${STORAGE_ACCOUNT}.file.core.windows.net`,
-    credentials,
-  );
+  const serviceClient = new ShareServiceClient(`https://${STORAGE_ACCOUNT}.file.core.windows.net`, credentials);
```

Comment:
`.file.core.windows.net` from a constant please.

Reply:
this was not changed/introduced as part of the PR. I'd like to keep the scope focused on eStore changes